app/mailers/petition_mailer.rb

```diff
@@ -165,6 +165,15 @@ def notify_creator_that_moderation_is_delayed(signature, subject, body)
       list_unsubscribe: unsubscribe_url
   end
 
+  def privacy_policy_update_email
+    @signature = params[:signature]
+
+    mail(
+      to: @signature.email,
+      subject: subject_for(:privacy_policy_update_email)
```

Comment:
`subject_for` uses translations, which might be overkill here given that it's a one-off email.